src/types/index.ts

```diff
@@ -939,6 +939,14 @@ export interface Permissions {
  */
 export type GroupPermissions = Pick<Permissions, 'transactions' | 'transactionGroups'>;
 
+/**
+ * This represents all Permission Groups of a Security Token
+ */
+export interface AllPermissionGroups {
```

Comment:
What about

```
/**
 * This represents all Permission Groups of a specific Security Token, separated by category
 */
export interface CategorizedPermissionGroups {
```

Reply:
What about just `PermissionGroups`, it looks like that won't conflict with anything.

I think Categorized is better than All, but `known` and `custom` feel like pretty broad categories to me.

Reply:
Pushed a commit with Categorized, but I can change it to just `PermissionGroups` also. 

Let me know what you prefer.

Reply:
`PermissionGroups` is better I think

Reply:
Updated to PermissionGroups